core-web/libs/sdk/react/src/lib/components/Row/Row.module.css

```diff
@@ -0,0 +1,5 @@
+.row {
+    display: grid;
+    grid-template-columns: repeat(12, 1fr);
+    gap: 1rem;
```

Comment:
should we remove this gap?

Reply:
I guess will see... tecnically they can disabled with class rows.